profile/main.go

```diff
@@ -341,11 +365,15 @@ func handler(request events.APIGatewayProxyRequest) (events.APIGatewayProxyRespo
 		if err != nil {
 			log.Fatalf("Failed to iterate: %v", err)
 		}
+		profileStatusCounter.totalProfilesChecked += 1
 		var userId string = doc.Ref.ID
 		var userUrl string
 		if str, ok := doc.Data()["profileURL"].(string); ok {
 			userUrl = str
 		} else {
+			profileStatusCounter.profilesSkippedDueToProfileURL += 1
+			profileStatusList.profilesSkippedDueToProfileURL = append(profileStatusList.profilesSkippedDueToProfileURL, userId)
```

Comment:
Can't we register only arrays and get the count from them.

Reply:
Makes sense.  We can do this.